config/initializers/devise.rb

```diff
@@ -13,25 +13,27 @@
 end
 
 GITHUB_OMNIAUTH_SETUP = lambda do |env|
+  env["omniauth.strategy"].options[:scope] = "user:email"
   env["omniauth.strategy"].options[:client_id] = Settings::Authentication.github_key
   env["omniauth.strategy"].options[:client_secret] = Settings::Authentication.github_secret
-  env["omniauth.strategy"].options[:scope] = "user:email"
 end
```

Comment:
This one was already here but I moved it up so all of the providers here have the scope appear first (consistency/readability)